src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -6,24 +6,55 @@
           <div class="ma-header">
             <span>Keyword Generator</span>
           </div>
-          <textarea v-model="inputValue" placeholder="Enter text here..." rows="6" class="input-textarea"></textarea>
+          <MaInput
```

Comment:
When using components in `template` we should always use `kebab-case` to comply with html standards.

```suggestion
          <ma-input
```

Reply:
I wrote MaInput since it was in the ActionKit [documentation ](https://mobileaction.github.io/action-kit/?path=/docs/components-mainput-textarea--docs). Will making it ma-input work?